src/api/Jalasoft.TeamUp.Projects.API/Controllers/GetInvitationsByResumeId.cs

```diff
@@ -0,0 +1,57 @@
+namespace Jalasoft.TeamUp.Projects.API.Controllers
+{
+    using System;
+    using System.IO;
+    using System.Net;
+    using System.Threading.Tasks;
+    using Jalasoft.TeamUp.Projects.Core.Interfaces;
+    using Jalasoft.TeamUp.Projects.Models;
+    using Jalasoft.TeamUp.Projects.ProjectsException;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Azure.WebJobs.Extensions.Http;
+    using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Attributes;
+    using Microsoft.Extensions.Logging;
+    using Microsoft.OpenApi.Models;
+    using Newtonsoft.Json;
+
+    public class GetInvitationsByResumeId
+    {
+        private readonly IInvitationsService invitationsService;
+
+        public GetInvitationsByResumeId(IInvitationsService invitationsService)
+        {
+            this.invitationsService = invitationsService;
+        }
+
+        [FunctionName("GetInvitationsByResumeId")]
+        [OpenApiOperation(operationId: "GetInvitationsByResumeId", tags: new[] { "Invitations" })]
+        [OpenApiParameter(name: "id", In = ParameterLocation.Path, Required = true, Type = typeof(int), Description = "The invitation identifier.")]
+        [OpenApiResponseWithBody(statusCode: HttpStatusCode.OK, contentType: "application/json", bodyType: typeof(Invitation), Description = "Successful response")]
```

Comment:
the body type should be an array of invitations

Reply:
done